.vitepress/config.js

```diff
@@ -9,8 +9,9 @@ export default defineConfig({
     },
 
     nav: [
+      { text: "Intro", link: "/script/", activeMatch: "/script/" },
```

Comment:
At the moment I don't intend on changing the name of the navbar navigation item to "Intro" (not that I'm against new names or even new sections entirely). I'm just not sure intro quite captures what I have in mind for this section. I also want to keep it in second position, as the primary goal of the website (at least for now) is to provide a complete encyclopedia of opcodes and short pages exploring them.

Reply:
sounds good, I'll revert